addons/binding/org.openhab.binding.jeelink/README.md

```diff
@@ -0,0 +1,135 @@
+# Jeelink Binding
+
+This binding integrates JeeLink USB RF receivers into OpenHAB 2. 
+
+## Introduction
+
+Binding should be compatible with JeeLink USB receivers and connected LaCrosse temperature sensors as well as EC3000 sensors.
+
+## Supported Things
+
+This binding supports:
+
+* JeeLink USB RF receivers as bridge
+* LaCrosse Temeperature Sensors connected to the bridge (running the LaCrosseITPlusReader sketch)
+* EC3000 Power Monitors connected to the bridge (running the ec3kSerial sketch)
+
+## Prerequisites
+
+The binding makes reads from the serial port and therefore needs openhab-transport-serial. Install the bundle in the **openhab2 console**
+by typing: 
+```
+feature:install openhab-transport-serial
+```
+
+At least on linux the user running openhab needs to have the necessary rights to access the serial devices. He has to be added to the
+group owning the device. Find out the group name in the **linux console** by typing
+```
+ls -al /dev/ttyUSB0
+```
+This should give output like
+```
+crw-rw---- 1 root dialout 188, 0 Okt  4 10:45 /dev/ttyUSB0
+```
+Now add the user to the group by typing:
+```
+sudo usermod -a -G groupname openhabuser
+```
+Make sure to replace _groupname_ and _openhabuser_ with the actual values. In this example group name is dialout. 
+
+## Binding Configuration
+
+Configuration of the binding is not needed. 
+
+## Thing discovery
+
+Only sensor discovery is supported, the thing for the USB receiver has to be created manually. Pay attention to use the correct serial port, as otherwise the binding may interfere with other bindings accessing serial ports.
+
+Afterwards, discovery reads from the USB receiver to find out which sensors are currently connected. It then creates a thing for every sensor for which currently no other thing with the same sensor ID is registered with the bridge. 
+
+## Thing configuration
+
+#### JeeLink USB RF receivers
+
+| Parameter         | Item Type    | Description
+|-------------------|--------------|------------
+| Port Name         | String       | Port to which the Jeelink is connected. See below
+| Sketch Name       | String       | Currently only LaCrosseITPlusReader is supported
+| Init Commands     | String       | A semicolon separated list of init commands that will be send to the Jeelink, e.g. "0a v" for disabling the LED.
+
+The Port Name has to be of the form:
```

Comment:
Logically it does not make sense to me to have a different thing for the same piece of hardware. What if I change this to support either normal serial ports (without prefix) and additionally allow to specify hostname and port by using the tcp:// prefix?